### ISSUE TITLE
Where to set API key?

### ISSUE THREAD

**Issue by Hickey8** [filippofinke/WordGPT#6]
I'm new to GPT API and I appreciate it a lot that you made such a superb add-in for Word. I beg to ask that when I  followed the instructions to insert the WordGPT add-in into Word. It showed 'add-in error' when I clicked WordGPT. I wanna know **where to set API key in the certain config** for me to use the api. Thanks a lot in advance.

**Comment by filippofinke** [comment]
Are you still getting an error?

**Comment by Hickey8** [comment]
> Are you still getting an error?

Yes ,it looks like this now becauese i don't where to set the API key
![image](https://user-images.githubusercontent.com/109857533/235168533-836e4093-6109-4e71-bc56-be4eb8b2ca82.png)

**Comment by qianwangwx** [comment]
I have the same question. I think it is because chatgpt version 1.0.0 or 1.1.0 need more technical support. It might work if you use the old version.
